webpack.config.js

```diff
@@ -0,0 +1,142 @@
+var webpack = require('webpack'),
+    path = require('path'),
+    autoprefixer = require('autoprefixer'),
+    Clean = require("clean-webpack-plugin"),
+    AssetsPlugin = require('assets-webpack-plugin'),
+    ExtractTextPlugin = require('extract-text-webpack-plugin'),
+    OptimizeCssAssetsPlugin = require('optimize-css-assets-webpack-plugin'),
+    cssnano = require('cssnano');
+
+var SAGE_ENV = process.env.SAGE_ENV || 'development',
+    webpackConfig;
+
+var sage = {
+  publicPath: '/app/themes/sage/dist/',
```

Comment:
this should be loaded from a config file or be built based on config file values